pkg/operator/k8sutil/prometheus_test.go

```diff
@@ -0,0 +1,27 @@
+package k8sutil
+
+import (
+	"os"
+	"path"
+	"testing"
+)
+
+func TestGetServiceMonitor(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
+	filePath := path.Join(gopath, "src/github.com/rook/rook/cluster/examples/kubernetes/ceph/monitoring/service-monitor.yaml")
+	_, err := GetServiceMonitor(filePath)
+	if err != nil {
```

Comment:
with testify you can use `assert.Nil(t, err)`.

Reply:
ack